<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YouXuanDockWidget</class>
 <widget class="QDockWidget" name="YouXuanDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>整治模式优选</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>298</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>乡名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>小刘</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>村名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>小黑</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>193</height>
      </rect>
     </property>
     <attribute name="label">
      <string>整治需求判断</string>
     </attribute>
     <widget class="QToolButton" name="btnKongxinStatus">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>查看</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>是否为空心村 ？</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>是否为需要整治 ？</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>是</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>否</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>是</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>否</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>空心化情况：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>193</height>
      </rect>
     </property>
     <attribute name="label">
      <string>村民改造意愿</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>59</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>总户数：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>同意改造户数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>反对改造户数：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>村民总体改造意愿：</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>强</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>弱</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>193</height>
      </rect>
     </property>
     <attribute name="label">
      <string>整治工作开展</string>
     </attribute>
     <widget class="QToolButton" name="btnSurveyForm">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>访问问卷调查</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnPictureCollet">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数据图件收集</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnSemiStructSurvey">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>半结构式访谈</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnSpecStatus">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>当地特殊情况</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnProblemStrategy">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>核心问题研判与对应策略选择</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>193</height>
      </rect>
     </property>
     <attribute name="label">
      <string>整治方案制定</string>
     </attribute>
     <widget class="QToolButton" name="btnInitStrategy">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>初步方案生成</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnFinalSolution">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>整治方案确定</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnPublicOpinion">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>村民意见收集</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnGovOpinion">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>政府部门要求</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
